.github/workflows/release.yaml

```diff
@@ -28,6 +28,12 @@ jobs:
       - name: Build changelog formatter
         run: pnpm --filter design-system-changelog-github build
 
+      # This will fail the build if something in the publish setup is not correct
+      # before changeset magic is starting to run
+      - name: Test publish step
+        id: publishtest
```

Comment:
Just for my own curiosity, for what the id is used?

Reply:
It's a leftover. You can assign IDs to steps so you can refer to outputs from that step. I first thought that I'd need to check the output of the test but the dry run just errors if anything is wrong and that error will stop the whole action, so it's actually not necessary there.